go.mod

```diff
@@ -1,60 +1,60 @@
 module github.com/attestantio/vouch
 
-go 1.22.7
+go 1.23.0
 
 toolchain go1.23.2
 
 require (
 	github.com/attestantio/go-block-relay v0.4.3
-	github.com/attestantio/go-builder-client v0.6.0
-	github.com/attestantio/go-eth2-client v0.24.0
-	github.com/aws/aws-sdk-go v1.55.5
+	github.com/attestantio/go-builder-client v0.6.1
+	github.com/attestantio/go-eth2-client v0.24.1-0.20250226000601-fb893f25a47a
```

Comment:
Is the version of go-eth2-client intended? It looks like it's possibly a branch

Reply:
Yep, that's not a good version.  I have pushed an update with a release version; thanks.